template/cordova/prebuild.js

```diff
@@ -30,20 +30,20 @@ module.exports = function patch (platform) {
     var url = require('url');
 
     var basejsSrcMap = {
-        '10': '/www/WinJS/js/base.js',
```

Comment:
Above we've had `'10.0'` - which is correct?

Reply:
Not sure. I don't use Windows.

And reverted the change. Its fixed now.